src/js/edu-benefits/education-wizard.js

```diff
@@ -41,12 +41,14 @@ function reInitWidget() {
 
   content.addEventListener('change', (e) => {
     const radio = e.target;
-    const otherChoice = radio.dataset.alternate;
-    const otherNextQuestion = container.querySelector(`#${otherChoice}`).dataset.nextQuestion;
-    if (otherNextQuestion) {
-      const otherNextQuestionElement = container.querySelector(`[data-question=${otherNextQuestion}]`);
-      closeStateAndCheckChild(otherNextQuestionElement, container);
-    }
+    const otherChoices = radio.dataset.alternate.split(' ');
```

Comment:
My understanding is that benefits are transferred from sponsors to dependents, but if that is mistaken, `update-non-dependent-transfer` would be clearer. Perhaps @goldenmeanie can advise, unless you're familiar with benefits be transferred to non-dependents.